PKHeX.Core.AutoMod/Teams/SmogonSetList.cs

```diff
@@ -58,12 +58,28 @@ private void LoadSetsFromPage()
             for (int i = 1; i < split1.Length; i++)
             {
                 var shiny = split1[i - 1].Contains("\"shiny\":true");
+                var level = split1[i - 1].Contains("\"format\":\"LC") ? 5 : 100;
+                if (!split1[i - 1].Contains("\"level\":0,") && split1[i - 1].Contains("\"level\":"))
+                {
+                    int.TryParse(split1[i - 1].Split(new[] {"\"level\":"}, StringSplitOptions.None)[1]
+                        .Split(',')[0], out level);
+                }
```

Comment:
Your level check here is wrong actually. Take this scenario. Lets say there are 3 sets within LC. The format isnt specified for each set but its specified as one of the keys in the json. So basically we have to look back till we find the first instance of the `format` key prior to the current split.

The current implementation will only be LC for the first LC set but wont find format LC in the previous split for set 2 and set 3. I'll try and commit to your branch likely tomorrow for a fix